src/pcre2_study.c

```diff
@@ -759,7 +759,7 @@ for (;;)
     }
   }
 
-  PCRE2_UNREACHABLE(); /* Control never reaches here */
+PCRE2_UNREACHABLE(); /* Control never reaches here */
```

Comment:
debated with myself about kicking these changes into a post cleanup patch, but I thought that since they were included since the first version and I am not clear yet on how the "audit" is going to be made or by whom it was safer to keep them for now to avoid anyone else having unnecessary conflicts or distracting noise.

Reply:
Yes, it's better to include them. Sorry that I was not used to the brace style for this project (still not used to it, but am getting there) when I included that extra indent.